content/source/docs/enterprise/api/configuration-versions.html.md

```diff
@@ -8,6 +8,102 @@ sidebar_current: "docs-enterprise2-api-configuration-versions"
 
 -> **Note**: These API endpoints are in beta and are subject to change.
 
+## List Configuration Versions
+
+| Method | Path           |
+| :----- | :------------- |
+| GET | /workspaces/:workspace_id/configuration-versions |
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/workspaces/ws-2Qhk7LHgbMrm3grF/configuration-versions
+```
+
+### Sample Response
+
+```json
+{
+    "data": [
```

Comment:
I think we've generally stuck to 2 spaces here, rather than 4. It'd be good to keep it consistent.